packages/@webex/internal-plugin-conversation/test/unit/spec/conversation.js

```diff
@@ -38,39 +40,36 @@ describe('plugin-conversation', () => {
       it('should add recipients to the payload if provided', () => {
         const {conversation} = webex.internal;
         const recipientId = 'example-recipient-id';
-        const expected = {items: [{id: recipientId, objectType: 'person'}]}
-        conversation.sendReaction = sinon.stub().returns(Promise.resolve())
-        conversation.createReactionHmac = sinon.stub().returns(Promise.resolve('hmac'))
-        
-        return conversation.addReaction({}, 'example-display-name', {}, recipientId)
-          .then(() => {
-            assert.deepEqual(conversation.sendReaction.args[0][1].recipients, expected);
-          });
+        const expected = {items: [{id: recipientId, objectType: 'person'}]};
```

Comment:
most of these changes are just whitespace/formatting changes, except for the added new describe block